contracts/Core/StakeManager.sol

```diff
@@ -436,6 +468,23 @@ contract StakeManager is Initializable, ACL, StakeStorage {
         return(numStakers);
     }
 
+    /// @return age of staker
+    function getAge(uint256 stakerId) external view returns(uint256) {
+        return stakers[stakerId].age;
+    }
+
+    /// @return maturity of staker
+    function getMaturity(uint256 stakerId) external view returns(uint256) {
```

Comment:
Can you please explain
Where Maturity is being used?

As per whitepaper
maturity = f(age)
reputation = f(maturity)
influence = reputation * stake

Reputation is Penalised

In current code
I see only age and influence
Age is penalized 
Influence = Age * Stake 

Please correct me If I am wrong

Reply:
Influence is used as weight.
Influence = stake * maturity

Reply:
oh then why in influence we have age in place of maturity ?
line no. 485

Reply:
yes fixed in https://github.com/razor-network/contracts/pull/218/commits/f8ccf9ce00ecce17230bed1d2a9199464358e940